Makefile

```diff
@@ -13,7 +13,7 @@ install: clean
 hash:
 	forge script script/HashData.s.sol
 
-sign\:%:
+sign\:%: hash
```

Comment:
when you run `sign` it supposed to run `hash` above? I didn't know this syntax.

Reply:
Exactly. I also use it on install: it builds the data directory first